### DISCUSSION TITLE
Exposing scale subresource for just component: node nodePools?

### DISCUSSION THREAD

**Discussion by BaconSandwiches** [Opster/opensearch-k8s-operator#429] [Q&A]
I was looking at hooking up KEDA to scale data nodes via the opensearchclusters CRD. My understanding is that this is not possible currently because there is no way to specify a static specReplicasPath or statusReplicasPath for the nodes component replicas? I know autoscaling is on the horizon in the roadmap, but was mainly curious if I could get something working in the interim for basic scaling.

Cheers.

**Comment by swoehrl-mw** [comment]
Hi @BaconSandwiches.
Your assumptions are correct. We do not support autoscaling with KEDA as our cluster CRD does not define the scale subresource. And as you guessed it would be hard as we have a replica property per nodepool so no clear way to define `specReplicasPath` and co.
So I'm afraid you will either have to find an external scaling solution that can modify arbitray paths in Custom Resources or wait for the autoscaling feature in the operator (no estimation when that will be done though).
